### PR TITLE
pgweb 0.11.8

### PR DESCRIPTION
**Pull request by chenrui333**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/pgweb.rb [hunk 1]
```diff
@@ -15,21 +15,15 @@ class Pgweb < Formula
...
   depends_on "go" => :build
-  depends_on "go-bindata" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "auto"
-    (buildpath/"src/github.com/sosedoff/pgweb").install buildpath.children
+    ldflags = %W[
+      -s -w
+      -X github.com/sosedoff/pgweb/pkg/command.BuildTime=#{Date.today}
```
Comment (bayandin) [inline Formula/pgweb.rb:22]: We can either use `time` (but then it should wait for the next release of homebrew/brew (See https://github.com/Homebrew/brew/pull/11659 and https://github.com/Homebrew/homebrew-core/pull/80608) or `Time.at(ENV["SOURCE_DATE_EPOCH"].to_i).utc.iso8601` instead right now.

```suggestion
      -X github.com/sosedoff/pgweb/pkg/command.BuildTime=#{time.iso8601}
```

Comment (chenrui333) [reply Formula/pgweb.rb:22]: I see, prety cool.

**Review by bayandin — DISMISSED** [review]

**Comment by bayandin** [comment]
Nice! Let's merge it right after the next release (3.2.2)

File: Formula/pgweb.rb [hunk 2]
```diff
@@ -15,21 +15,15 @@ class Pgweb < Formula
...
-  depends_on "go-bindata" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "auto"
-    (buildpath/"src/github.com/sosedoff/pgweb").install buildpath.children
+    ldflags = %W[
+      -s -w
+      -X github.com/sosedoff/pgweb/pkg/command.BuildTime=#{time.iso8601}
+      -X github.com/sosedoff/pgweb/pkg/command.GoVersion=#{version}
```
Comment (alebcay) [inline Formula/pgweb.rb:23]: Shouldn't this be the version of the Go runtime being used in build, not the version of `pgweb`?

There's also a variable `GitCommit`, not sure if that needs to be set.

Comment (chenrui333) [reply Formula/pgweb.rb:23]: yeah, I am aware of that GitCommit thing, but I dont know why that is useful (I feel the version and builtBy would be enough)

Comment (alebcay) [reply Formula/pgweb.rb:23]: Okay. But regarding `GoVersion`, maybe that should be `Formula["go"].version` or something like that?

Comment (chenrui333) [reply Formula/pgweb.rb:23]: you are absolutely right, I dont know why upstream is using GoVersion, let me file a PR asking about it. 😂

Comment (chenrui333) [reply Formula/pgweb.rb:23]: I see, https://github.com/sosedoff/pgweb/blob/4de356cb1d47d7407169d75a710520c6cd184fb7/pkg/command/version.go#L4-L5, my bad.

File: Formula/pgweb.rb [hunk 3]
```diff
@@ -15,21 +15,15 @@ class Pgweb < Formula
...
-  depends_on "go-bindata" => :build
 
   def install
-    ENV["GOPATH"] = buildpath
-    ENV["GO111MODULE"] = "auto"
-    (buildpath/"src/github.com/sosedoff/pgweb").install buildpath.children
+    ldflags = %W[
+      -s -w
+      -X github.com/sosedoff/pgweb/pkg/command.BuildTime=#{time.iso8601}
+      -X github.com/sosedoff/pgweb/pkg/command.GoVersion=Formula["go"].version
```
Comment (bayandin) [inline Formula/pgweb.rb:23]:
```suggestion
      -X github.com/sosedoff/pgweb/pkg/command.GoVersion=#{Formula["go"].version}
```

**Comment by nandahkrishna** [comment]
`3.2.2` was released so we should be able to merge this.

**Review by nandahkrishna — APPROVED** [review]

**Pull request closed without merging**